applications/blogs/backend/src/main/resources/libs/kyanite/blogs/components/quote/dialog/.content.json

```diff
@@ -0,0 +1,50 @@
+{
+  "sling:resourceType": "wcm/dialogs/dialog",
+  "tabs": {
+    "sling:resourceType": "wcm/dialogs/components/tabs",
+    "generalTab": {
+      "sling:resourceType": "wcm/dialogs/components/tab",
+      "label": "General",
+      "quoteText": {
+        "sling:resourceType": "wcm/dialogs/components/richtext",
+        "name": "quoteText",
+        "label": "Quote text",
+        "required": true,
+        "ws:disallowedContext": ["edit:panel"],
+        "configuration": "kyanite/common/components/common/rte/configurations/default"
+      },
+      "authorContainer": {
+        "sling:resourceType": "wcm/dialogs/components/container",
+        "authorName": {
+          "sling:resourceType": "wcm/dialogs/components/textfield",
+          "name": "authorName",
+          "label": "Author name",
+          "required": true
+        },
+        "authorCompany": {
+          "sling:resourceType": "wcm/dialogs/components/textfield",
+          "name": "authorCompany",
+          "label": "Author company"
+        },
+        "authorPhotoPath": {
+          "sling:resourceType": "wcm/dialogs/components/pathpicker",
```

Comment:
I strictly followed the design, so the only thing needed from this property was a valid picture to display. What freedoms do we want to give to authors?

Reply:
But, I think for authors it's easier to drag and drop the image from assets then finding an image by it's path?

Reply:
And maybe to avoid misunderstanding, I think about using:
```
"sling:resourceType": "wcm/dialogs/components/assetreference",
"mimeTypes": ["image/*"],
```

Reply:
Maybe `kyanite/common/components/common/imagesource` will fit better?
<img width="322" alt="image" src="https://github.com/websight-io/kyanite/assets/13979207/f94abf31-eeab-42ab-bdc4-8b939de14808">

Reply:
The 'Link' imageSource type suffers from [side panel extra save-properties request](https://teamds.atlassian.net/browse/WS-3150) because of pathpicker - authors may lose their work - so I hid the field from sidepanel, same as pathpicker previously. When the issue is resolved we will need to inspect all the properties with ws:disallowedContext: ["editPanel"] and probably unhide some.